backend/controllers/bulkAppointmentsClashes.js

```diff
@@ -67,6 +67,7 @@ const bulkAppointmentsClashesFactory = (elite2Api, logError) => {
 
   const post = async (req, res) => {
     const {
+      data,
```

Comment:
didn't know you could refer to the same thing more than once when destructuring 🤓